src/components/form/form-button/form-button.spec.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import { rootTagTest } from '../../../utils/helpers/tags/tags-specs/tags-specs';
+import FormButton from './form-button.component';
+import Button from '../../button';
+
+const renderFormButton = (props, render = shallow) => {
+  return render(
+    <FormButton
+      { ...props }
+    />
+  );
+};
+
+describe('FormButton', () => {
+  let wrapper, clickFunction, button;
+
+  describe.each(['save', 'cancel'])(
+    'the primary action buttons',
+    (name) => {
+      beforeEach(() => {
+        wrapper = renderFormButton({
+          formButtonName: name,
+          [`${name}ButtonProps`]: { className: 'my-custom-button-class' }
+        });
+      });
+
+      it('renders a custom class if provided', () => {
+        expect(wrapper.find('.my-custom-button-class').exists()).toBeTruthy();
+      });
+
+      it('renders the default text', () => {
+        button = wrapper.find(Button);
+        const buttonName = name.charAt(0).toUpperCase() + name.slice(1);
+        expect(button.prop('children')).toEqual(buttonName);
+      });
+
+      it('renders custom text when provided', () => {
+        wrapper = renderFormButton({
+          formButtonName: name,
+          [`${name}Text`]: `This is a ${name} button`
+        }, shallow);
```

Comment:
I don't think `shallow` is needed here, is it? (Since the `render` parameter of `renderFormButton()` has a default value of `shallow`)

Reply:
Yep good spot